templates/dotnet/src/Appwrite/Client.cs.twig

```diff
@@ -36,7 +36,12 @@ namespace {{ spec.title | caseUcfirst }}
             this.headers = new Dictionary<string, string>()
             {
                 { "content-type", "application/json" },
-                { "x-sdk-version", "{{spec.title | caseDash}}:{{ language.name | caseLower }}:{{sdk.version}}" }
+                { "x-sdk-version", "{{spec.title | caseDash}}:{{ language.name | caseLower }}:{{sdk.version}}" } {% if spec.global.defaultHeaders | length > 0 %},{% endif %}
+
+        {% for key,header in spec.global.defaultHeaders %}
+        {"{{key}}", "{{header}}"}{% if not loop.last %},{% endif %}
```

Comment:
`{ "{{key}}", "{{header}}" }{% if not loop.last %},{% endif %}`

Add spaces to fit previous code 🙂

Reply:
+1